.github/workflows/release.yml

```
@@ -0,0 +1,164 @@
+          echo '```' >> CHANGELOG.md
+          echo "" >> CHANGELOG.md
+          echo "## Railway Deployment" >> CHANGELOG.md
+          echo "" >> CHANGELOG.md
+          echo "This release is optimized for deployment on Railway with automatic database migrations." >> CHANGELOG.md
+          echo "" >> CHANGELOG.md
+          echo "## Environment Variables" >> CHANGELOG.md
+          echo "" >> CHANGELOG.md
+          echo "See the [README](README.md) for required environment variables." >> CHANGELOG.md
+
+      - name: Create Release
+        uses: softprops/action-gh-release@v1
```

actionlint (1.7.10)

the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)